### DISCUSSION TITLE
그룹#1 회고

### DISCUSSION THREAD

**Discussion by godrm** [codesquad-members-2022/swift-photoframe#136] [Show and tell]
여기에 각자 이번주 회고를 기록해주세요.
- 학습하면서 잘했던 점, 좋았던 점은 무엇인가요? (다음주에도 계속 유지해보세요)
- 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요 - 그래서 어떻게 개선할 수 있을까요? (다음주에는 개선해봐요)
- 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? (다음주에 시도해봐요)
- 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!

**Comment by Jeeehee** [comment]
* 하루도 거르지 않고 학습정리를 했고, 학습 내용과 연관해 코드로 많은 시도들을 해보았습니다.

* 어느 순간 혼자 해결하려는 노력보다, 동료들에게 질문하고 의존하는 성향이 더 커졌습니다.
주 후반에는 스스로 해결해보려는 노력을 더 해보았습니다.
다음주에도 의존보단 자립을 위해 노력하겠습니당 🤜🏻

* 대부분의 주제가 새로웠고, 더 많은 예제들을 시도해보고싶습니다.
다음주에는 미션 진도에 더 연연하지 않고, 예제들을 충분히 시도해 학습에 더 많은 초점을 두겠습니다.

* 모든 iOS 동료들에게 동기부여를 받았습니다.
특히 싱하와 솔, 피그백이 혼자라면 해결 못했을 문제들을 도와주셨습니당! 😇

**Reply by BumgeunSong** [reply]
다른 조이지만 댓글 달러 왔습니다 ㅋㅋ
스크린샷을 보며 디자이너 Jee의 클라스를 느꼈습니다

**Reply by bibi6666667** [reply]
ㄴ22.. jee의 앱디자인 특강 원해요

**Reply by Jeeehee** [reply]
@BumgeunSong  @bibi6666667  사랑해여 여러분

**Reply by sungju-kim** [reply]
jee 디자인한거보고 제 친구리스트에서 빠졌습니다 😒

**Reply by Damagucci-Juice** [reply]
ㄴ2222222222

**Reply by JasonLee0223** [reply]
Jee 디자인 대박... 한 주 고생했어요~👊🏻👊🏻 일요일 게더 스터디룸에서 봅시다! ㅋㅋㅋㅋ👍🏻

**Reply by shingha1124** [reply]
디자인보고 제 앱이 부끄러웠습니다

**Reply by BumgeunSong** [reply]
![](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTeO7g6Mk4kwL266UBYRZM61IlYVVF5YicdGA&usqp=CAU)

**Reply by LeeJoobang** [reply]
지이이이이이이이이이 앱개발에 특화되었음...넘사벽

**Comment by yeaju-Choi** [comment]
- 학습을 하며 정리하는 습관이 없었는데, PR과 Readme를 작성하며 공부했던 내용을 복습할 수 있어 좋았다. 실습을 하며 다른 방법은 어떤 것이 있는지 고민하고, 왜 이방법을 썼는지 생각하며 학습하는게 어려웠지만 제대로 알아가는것 같아 좋았다. 

- 금요일까지 미션을 다 완료하고 싶었지만.. 생각보다 시간이 오래걸렸다. 하루 목표를 구체적으로 세워야 겠다. 

- 그룹리뷰때 ibdesinable, type annotation/type 추론 등의 키워드가 나와서 다음주에는 공부해봐야겠다. 다음주에는 미션 구현기능에 초점만 두지 않고 디자인적으로 좀더 이쁘게 만들어보고 싶다. 

- 모두에게 도움을 받았다. 생각해보지 못한 문제점이나, 키워드들을 말씀을 해주셔서 새로운 정보들을 많이 얻을 수 있었다. :)

**Reply by Jeeehee** [reply]
체즈 고생해써여 ❤️

**Reply by yeaju-Choi** [reply]
@Dihee  😘

**Comment by sanghyeok-kim** [comment]
### 잘했거나 좋았던 점
- Swift 문법만 하다가 처음으로 iOS 프로젝트를 시작했는데, 처음에 스스로 걱정했던 것 보다는 훨씬 잘 해내고 있는 것 같습니다.

### 아쉽거나 부족했던 점
- 아직 Git과 완전히 친해지지는 못한 것 같습니다. 과정이 끝날 때 쯤엔 꼭 친해져있으면 좋겠습니다.

### 새롭게 알게된 점
- View LifeCycle
- Scene과 Window, View와 ViewController에 대한 기본적인 개념 정립

### 도전 목표
- 주말동안 이번 프로젝트를 코드위주로 다시 만들어볼 생각입니다. 

### 도움을 준 사람들
- Git에서 문제가 생겼을 때 오랜 시간 같이 고민해주신 dale과 shingha에게 특히 감사를 전하고 싶습니다.
- 그리고 한 주 동안 많은 도움을 주신 1조 여러분 모두 감사드립니다!!!
- (+ 무엇보다 방청객으로 참여해주셨던 다마구찌 장영진씨에게 가장 특별한 감사를 드립니다)
- (++ 무엇무엇보다 앱 생명주기와 뷰 생명주기에 대한 이해를 친절하게 도와주신 로사에게 아주 가장 최고 베스트 특별한 감사를 드립니다)

**Reply by Damagucci-Juice** [reply]
아니 ... 제가 소스코드 오토 세이브 기능을 발표하실때 같이 자리해드렸는데 정말 섭섭합니다. 언급이 없다뇨..

**Reply by BumgeunSong** [reply]
Mase가 잘못했네

**Reply by sanghyeok-kim** [reply]
구찌 방청객으로 참여해주셔서 정말 감사했습니다!^_^

**Reply by Damagucci-Juice** [reply]
하하 네 도움을 준 사람들에 추가 낭낭하게 부탁드립니다 ^^

**Reply by sungju-kim** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 구찌 ㅋㅋㅋㅋㅋㅋㅋ

**Reply by Jinsujin** [reply]
메이스 너무하네 나는요....?!

**Comment by SongTaehwan** [comment]
- 전날 미리 학습할 키워드를 골라두어서 스크럼할 때 공유하기 편했고 학습하는 중간 확인하면서 미션 주제와 관련있는 내용에만 집중할 수 있었습니다.
- 미션을 끝내야할 일처럼 생각해 조급한 마음을 가지고 있었습니다. 미션은 성장해나아가야할 길을 제시하는 하나의 이정표라고 생각하고 학습에 더 집중하겠습니다.
- 더 다양한 UI 컴포넌트들을 활용해보는 연습을 해보고자합니다.
- 새벽까지도 올라오는 PR 을 보면서 반성하게 되고 자극받기도 했습니다. 조원들에게서 스스로 생각해보지 못한 좋은 의문점들이나 유용한 팁을 얻을 수 있어서 많은 도움을 받았습니다.

**Comment by rising-jun** [comment]
- 그동안 놓치고 지나갔던 학습 키워드들에 대해 공부하게 되어 좋았습니다.

- 실패에 대해 너무 두려워했던것 같습니다.(git conflict) 새로운 방법에 도전하지 않고 계속 cli와 xcode만 사용하여 깃을 공부하였습니다. 
이를 개선하기 위해 gitkraken이나 sourcetree를 사용하여 도전해보고 실패를 두려워하지 않겠습니다!

- 계속 개선하려 노력중인 것은 디자인api가이드 내용처럼 "명확한" 이름을 짓는 것입니다. 다른 사람들의 코드를 많이 읽어보고 혼자서도 곰곰히 고민해보며 이를 개선할 예정입니다.

- 같은 조원분들과 계속 공유하며 할 수 있어 너무 좋았습니다. 특히 새벽까지 저의 많은 질문들을 답해주신 rosa에게 감사합니다!

**Reply by Damagucci-Juice** [reply]
에보니 갓보니 킹보니 야야야 에코장인 에보니 야야야

**Reply by sungju-kim** [reply]
야경 동반자 ^^

**Comment by Hansolkkim** [comment]
- 학습 정리를 할 포인트들이 많아(아는게 적어) 그저 과제 수행을 목표로 하기 보다는 과제 수행에 필요한 내용을 나름대로 깊이있게 공부하여 학습 정리를 할 수 있어서 좋았습니다. 그리고 일주일치 내용이 월요일에 모두 공개되어, 공부할 내용을 미리 짚어보고 일정을 짜기에 수월해서 좋았습니다.

- 기본적으로 혼자 고민한 뒤에 동료들과 얘기를 나누어 풀어봐야 하는데, 잘 하는 동료가 있는 상황에서는 혼자 고민하는 시간 거의 없이 물어보고 답을 얻으려는 경향이 생겼던것 같습니다. 제 성장을 위해서는 혼자 고민하고 풀어보려 노력하는 과정이 좀 더 필요하다고 느꼈습니다.

- Step8 추가학습거리를 진행하면서 여러 시스템 컨트롤러에 대해 알 수 있었는데, 구현해보는 단계까지는 다다르지 못했습니다. 다음 주에는 새로운 시스템 컨트롤러에 대해서도 구현해보는 시간을 가져보고 싶습니다.

- 1조 조원분들께 감사의 말씀을 드리고 싶습니다. 같은 과제를 비교적 비슷한 학습 단계에서 진행하다보니, 비슷한 구간에서 헤메고 막히는 경우가 많았기 때문에 질문도 부담없이 할 수 있었고 많은 도움을 받을 수 있었습니다. 감사합니다!~!

**Reply by BumgeunSong** [reply]
고생한 쏠에게 쏠 한잔 cheers
![](https://jjalbang.today/files/jjalboxthumb/2021/12/102_10196.jpg)

**Comment by AhnSangHee** [comment]
### 잘했던 점, 좋았던 점
- 미션을 해결하면서 몰랐던 것에 대해서 계속 파고 들어가면서 새로운 것들을 학습할 수 있었던 점
- 지금까지 코드로 화면을 구현해본 적이 없었는데 코드로 구현해 보았던 점

### 아쉬웠던 점, 부족했던 점
- 학습할 것이 많아 미션 해결 속도가 느렸던 것 같아 아쉽습니다. 그러다보니 스크럼 때 말했던 목표를 이루지 못한 경우가 많았습니다.
- 계속 README를 쓰다보면 README가 너무 방대해지는 것 같아서 적절하게 쓸 수 있도록 해야할 것 같습니다.

### 다음주에 도전해볼 것
- 스크럼 때 목표를 적절하게 세운 후 그날의 목표 달성하기, 정리 꾸준히 잘하기

### 학습하는데 도움을 주신 분들
- 1조 모든 분들 그리고 늦은 시간에 게더에서 도와주신 shingha 너무 감사합니다..!

**Reply by BumgeunSong** [reply]
모든 조에서 싱하 칭찬이 나오네요 ㅎㅎ

**Comment by piggyse** [comment]
### 학습하면서 잘했던 점, 좋았던 점은 무엇인가요?
 - CS는 억지로 집중했다면 앱개발은 뭔가 재미를 붙여서 집중을 더 잘한것 같다.
 - git이라는 것을 언젠간 해야지하고 미뤄뒀었는데 미뤄둔 숙제를 조금이나마 한것 같아서 좋았다.
 - 다른분들의 PR을 확인하면서 내가 미처 생각하지 못한 부분도 확인하고 얻을 수 있어서 좋았습니다.

### 학습하면서 스스로 아쉬웠던 점, 부족했던 점은 무엇인가요
- 다른 분들의 PR에서 귀찮아서 애써 무시했던 내용을 조사하신 걸 보고 좀 더 끈기를 가지지 못했던 점이 아쉽습니다.

### 새롭게 알게 되거나 시도하지 못했지만, 다음주에 도전해볼건 무엇인가요? 
- 앱이 실행되고 View가 만들어지는 과정이 궁금해서 찾아봤는데 용어가 너무 어려워서 미뤄놨습니다. 다음주에 여유가 되면 하고싶습니다

### 학습하는 데 조금이라도 도움이 된 사람이 있다면 칭찬해주세요!
- 다들 바쁘실텐데 내색 안하고 도와주셔서 감사했습니다! 특히 구찌가 깃 도와주셔서 너무 감사했습니다!
